app/coffee/Notifications.coffee

```diff
@@ -81,6 +81,13 @@ module.exports = Notifications =
 			key: notification_key
 		db.notifications.remove searchOps, {justOne: true}, callback
 
+	removeNotificationIpMatcher: (user_id, university_id, callback)->
```

Comment:
Same comment as on the other PR, it feels like this might be more natural as `...ByIpMatcher`